### ISSUE TITLE
Double tap to USB and OTA

### ISSUE THREAD

**Issue by tannewt** [adafruit/Adafruit_nRF52_Bootloader#78]
Let's have the bootloader do DFU and USB loading all at the same time. That way we don't need to teach a separate process for loading code OTA.

**Comment by hathach** [comment]
yeah, good idea. We can also remove the 2nd pin for OTA as well. Currently either OTA or serial dfu run at the same time. Let's me check to enable them both (there is a middle layer from nordic to handle packet from either transportation).
